packages/react/src/components/legacy/LegacySelect/SingleSelect.stories.mdx

```diff
@@ -132,7 +132,10 @@ export const Template = (args = {}) => (
 
 # Select (enkel)
```

Comment:
Should be `LegacySelect`

Reply:
Fixed! There was a few more so did a search and replace in the mdx files: https://github.com/digdir/designsystem/pull/1250/commits/4352b57ef1e9cbec442ae36315ded9459f801080